tests/kafkatest/services/zookeeper.py

```diff
@@ -107,15 +107,15 @@ def alive(self, node):
     def stop_node(self, node):
         idx = self.idx(node)
         self.logger.info("Stopping %s node %d on %s" % (type(self).__name__, idx, node.account.hostname))
-        node.account.kill_process("zookeeper", allow_fail=False)
+        node.account.kill_java_processes("quorum.QuorumPeerMain", allow_fail=False)
```

Comment:
Why does only this one take a package prefix?

Reply:
I was being lazy here and just giving a pattern which I felt would be unique for identifying zookeeper.  However, I think it's better to just implement `java_class_name` as you suggested below, and use that.